src/theory/arith/nonlinear_extension.cpp

```diff
@@ -3256,6 +3295,11 @@ std::vector<Node> NonlinearExtension::checkTranscendentalTangentPlanes()
   for (std::pair<const Kind, std::map<Node, Node> >& tfs : d_tf_rep_map)
   {
     Kind k = tfs.first;
+    if (k == PI)
+    {
+      // we do not use Taylor approximation for PI currently
```

Comment:
create an issue?

Reply:
This is a resolved issue, a while ago I did experiments to test how fast a Taylor series converges for PI, and it was extremely slow. I can document this more.